pkg/controller/cronjob/cronjob_controllerv2.go

```diff
@@ -48,6 +48,7 @@ import (
 	"k8s.io/kubernetes/pkg/controller"
 	"k8s.io/kubernetes/pkg/controller/cronjob/metrics"
 	"k8s.io/utils/pointer"
```

Comment:
I see both k8s.io/utils/pointer and k8s.io/utils/ptr.

Are both needed?

Reply:
From what I can tell, the `ptr` package should replace the `pointer` package.

Reply:
bowei,  both are needed. you could find "pointer.StringEqual" still used in this file and there is another PR is working on this:

https://github.com/kubernetes/kubernetes/pull/120629